defmt/src/impls/core_/net.rs

```diff
@@ -0,0 +1,69 @@
+use core::net;
+
+use super::*;
+
+impl Format for net::AddrParseError {
+    fn format(&self, fmt: Formatter) {
+        crate::write!(fmt, "AddrParseError(_)");
+    }
+}
+
+impl Format for net::Ipv4Addr {
+    fn format(&self, fmt: Formatter) {
+        crate::write!(
+            fmt,
+            "{}.{}.{}.{}",
+            self.octets()[0],
+            self.octets()[1],
+            self.octets()[2],
+            self.octets()[3]
+        );
+    }
+}
+
+impl Format for net::SocketAddrV4 {
+    fn format(&self, fmt: Formatter) {
+        crate::write!(fmt, "{}:{}", self.ip(), self.port());
+    }
+}
+
+impl Format for net::Ipv6Addr {
+    fn format(&self, fmt: Formatter) {
+        crate::write!(
+            fmt,
+            "{:x}:{:x}:{:x}:{:x}:{:x}:{:x}:{:x}:{:x}",
+            self.octets()[0],
+            self.octets()[1],
+            self.octets()[2],
+            self.octets()[3],
+            self.octets()[4],
+            self.octets()[5],
+            self.octets()[6],
+            self.octets()[7]
+        );
+    }
+}
```

Comment:
`Ipv6Addr::octets(&self)` returns a `[u8; 16]` ([docs](https://doc.rust-lang.org/stable/std/net/struct.Ipv6Addr.html#method.octets)). Are you purposefully just using the first eight?

Reply:
Whoops, that's a mistake on my part, good catch, thanks!